README.md

```diff
@@ -68,6 +68,30 @@ devicefarm {
     // Optional. Defaults to "Top Devices"
     devicePool "My Device Pool Name"
     
+    projectName "My Project" // required: Must already exists.
```

Comment:
This needs to be rebased with master (I think).

You've got duplicate `projectName` and `devicePool` entries with slightly different format so they're being duplicated in the `README` twice.